config/bidderinfo.go

```diff
@@ -0,0 +1,88 @@
+package config
+
+import (
+	"fmt"
+	"io/ioutil"
+	"strings"
+
+	"github.com/prebid/prebid-server/openrtb_ext"
+	"gopkg.in/yaml.v2"
+)
+
+// BidderInfos contains a mapping of bidder name to bidder info.
+type BidderInfos map[string]BidderInfo
+
+// BidderInfo is the maintainer information, supported auction types, and feature opts-in for a bidder.
+type BidderInfo struct {
+	Enabled                 bool              // copied from adapter config for convenience. to be refactored.
+	Maintainer              *MaintainerInfo   `yaml:"maintainer"`
+	Capabilities            *CapabilitiesInfo `yaml:"capabilities"`
+	ModifyingVastXmlAllowed bool              `yaml:"modifyingVastXmlAllowed"`
+	Debug                   *DebugInfo        `yaml:"debug,omitempty"`
+}
+
+// MaintainerInfo is the support email address for a bidder.
+type MaintainerInfo struct {
+	Email string `yaml:"email"`
+}
+
+// CapabilitiesInfo is the supported platforms for a bidder.
+type CapabilitiesInfo struct {
+	App  *PlatformInfo `yaml:"app"`
+	Site *PlatformInfo `yaml:"site"`
+}
+
+// PlatformInfo is the supported media types for a bidder.
+type PlatformInfo struct {
+	MediaTypes []openrtb_ext.BidType `yaml:"mediaTypes"`
+}
+
+// DebugInfo is the supported debug options for a bidder.
+type DebugInfo struct {
+	Allow bool `yaml:"allow"`
+}
+
+// LoadBidderInfoFromDisk parses all static/bidder-info/{bidder}.yaml files from the file system.
+func LoadBidderInfoFromDisk(path string, adapterConfigs map[string]Adapter, bidders []string) (map[string]BidderInfo, error) {
+	reader := infoReaderFromDisk{path}
+	return loadBidderInfo(reader, adapterConfigs, bidders)
+}
+
+func loadBidderInfo(r infoReader, adapterConfigs map[string]Adapter, bidders []string) (map[string]BidderInfo, error) {
+	infos := map[string]BidderInfo{}
+
+	for _, bidder := range bidders {
+		data, err := r.Read(bidder)
+		if err != nil {
+			return nil, err
+		}
+
+		var info BidderInfo
```

Comment:
variable scope can be a little tricky. I suspect that this line will allocate a new `info` on each loop, but I can see an argument that `info` is scoped just once within the loop. I think using `info := BidderInfo{}` to force allocating a new structure each loop may be safer, and make it more clear that a new info structure is being allocated each loop.